manifests/01-registry-credentials-request-openstack.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: cloudcredential.openshift.io/v1
+kind: CredentialsRequest
+metadata:
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: openstack-openshift-image-registry
```

Comment:
```
azure-openshift-ingress            16m
cloud-credential-operator-iam-ro   17m
openshift-image-registry           17m
openshift-ingress                  17m
openshift-machine-api              17m
openshift-machine-api-azure        16m
```

@adambkaplan do we have a recommended pattern for these names?

Reply:
looks like we don't have a uniform standard, but I'd stick with `openshift-image-registry-<platform>`

Reply:
If we rename the original AWS request, hopefully the upgrade tests pass...